internal/slicer/slicer.go

```diff
@@ -180,6 +180,13 @@ func Run(options *RunOptions) (*Report, error) {
 					globbedPaths[extractInfo.Path] = append(globbedPaths[extractInfo.Path], relPath)
 					addKnownPath(relPath)
 				}
+
+				// Do not add paths with "until: mutate"
+				info, ok := pathInfos[extractInfo.Path]
+				if !ok || info.Until == setup.UntilMutate {
```

Comment:
If the path is not in pathInfos should it be an error?

Reply:
Since the path already exists in `slice.Contents` above, it should also be in `pathInfos` with only one exception where the listed `arch`-es do not include the `chisel cut --arch`. But then again, `deb.Extract` would not be called for that path for the very same reason. So yeah, we could throw an `internal error` if it does not exist.

Reply:
Added in recent commits.